css/index.css

```
@@ -818,3 +818,482 @@ body {
+    width: 209.51px;
+    height: 169.05px;
+  }
+
+  .most-popular__name {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 22px;
+    text-align: left;
+    color:#716C6C;
+    margin-right: 70px;
+  }
+
+  .most-popular__total {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 22px;
+  }
+
+  .most-popular__number {
+    font-size: 14px;
+  }
+}
+
+/* Mobile-first right*/
+.most-popular__right {
+  border-radius: 10px;
+  padding: 10px 10px 20px 10px;
+  position: relative;
+  overflow: hidden;
+  background-size: cover;
+  background-position: center center;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: #D14D41 solid 2px;
+}
+
+
+
+.most-popular__right-img {
+  width: 100%; 
+  height: auto; 
+}
+
+.most-popular__right-text {
+  align-self: flex-start;
+  display: flex;
+  flex-direction: column;
+}
+
+.most-popular__do {
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: left;
+  color: #A6A6A6;
+}
+
+.most-popular__right-total {
+  display: flex;
+  justify-content: space-between;
+  font-size: 25px;
+  width: 100%;
+}
+
+.most-popular__right-name {
+  font-size: 20px; 
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: -1%;
+  text-align: left;
+  color: #6C6B6A;
+  margin-top: 10px;
+  margin-bottom: 20px;
+}
+
+.most-popular__right-price {
+  font-size: 24px; 
+  font-weight: 700;
+  line-height: 30px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  color: #292727;
+}
+
+.most-popular__prioridad {
+  width: 14.76px;
+  height: 14.76px;
+  margin-left: 15px;
+}
+
+.most-popular__rigiht-entrega {
+  color: #D14D41;
+  font-size: 14.76px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 18.55px;
+  text-align: left;
+  background-color: #FAEDEC;
+  width: auto; 
+  padding: 7.38px 12px;
+  gap: 4px;
+  border-radius: 4.92px 0px 0px 4.92px;
+  align-items: center;
+  display: flex;
+
+}
+
+.most-popular__bord {
+  width: 100%; 
+  border: 0.5px solid #D8D8D8;
+  margin-top: 15.35px;
+}
+
+
+.most-popular__bord-botton {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.most-popular__bord-button{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  width: 268.49px;
+  height: 72.09px;
+  gap: 6px;
+  border-radius: 8px;
+  color: #FFFFFF;
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 17.6px;
+  letter-spacing: 6%;
+  margin-top: 13px;
+  margin-right: 20px;
+  background-color: #D14D41;
+}
+
+
+.arrow {
+  width: 16px;
+  height: 16px;
+  margin-bottom: -3px;
+}
+
+.most-popular__right-method {
+  align-self: flex-start;
+  margin-top: 45px;
+}
+
+.most-popular__right-shipping,
+.most-popular__right-removal {
+  display: flex;
+  align-items: center;
+}
+
+.most-popular__right-shipping .most-popular__right-close,
+.most-popular__right-removal .most-popular__right-check {
```

Anidacion innecesaria
```suggestion
.most-popular__right-close,
.most-popular__right-check {
```